process/block/displayBlock.go

```diff
@@ -95,6 +66,7 @@ func (txc *transactionCounter) displayLogInfo(
 ) {
 	dispHeader, dispLines := txc.createDisplayableShardHeaderAndBlockBody(header, body)
 
+	// TODO: Why read lock acquired for setting a value?
```

Comment:
The mutex is for txc.totalTxs which is the common resource consumed (read/write) in this file

Reply:
Removed TODO.